tests/foreman/api/test_subnet.py

```diff
@@ -497,6 +496,34 @@ def test_positive_list_parameters(self):
         :expectedresults: The satellite should display all the subnet
             parameters
         """
+        parameter = [{'name': gen_string('alpha'),
+                      'value': gen_string('alpha')}]
```

Comment:
Can you make it just a dict, and wrap it around list when you submit it?
This way your assertions can be written as:
```
        self.assertEqual(org_subnet.subnet_parameters_attributes[0]['name'],
                         parameter['name'])
```
Or even create two variables - `parameter_name` and `parameter_value`.

Reply:
Done.